_sass/addon/commons.scss

```diff
@@ -106,8 +106,22 @@ blockquote {
   color: var(--blockquote-text-color);
 }
 
+/* Pretty Keyboard Keys
+usage: <kbd>⌘ Cmd</kbd>+<kbd>⇧ Shift</kbd>+<kbd>.</kbd> */
+
 kbd {
+  background-color: #ecede5;
+  border-radius: .1rem;
+  box-shadow: 0 0.1rem 0 0.05rem #686d59,0 0.1rem 0 #686d59,0 -0.1rem 0.2rem #fff inset;
+  color: #2d3329;
+  display: inline-block;
+  font-size: .75em;
+  font-family: var(--md-code-font-family),SFMono-Regular,Consolas,Menlo,monospace;
```

Comment:
Did you copy the CSS from elsewhere? Because an undefined variable `--md-code-font-family` is used here.

Anyway, thanks for the PR, but I don’t plan to merge it, because the default black keyboard is beautiful enough, you don’t need to change it to white. However, adding shadows to `<kbd>` and changing the font is worth considering. Later I will complete this task.

Reply:
Thanks, understood.

Reply:
I was building this for mkdocs as I also manage an internal documentation site using that and forgot to remove the extra code.

Reply:
@cotes2020  Thanks for taking my idea into consideration. I love how you implemented it.